src/coins.js

```diff
@@ -3,56 +3,151 @@ const typeforce = require('typeforce')
 
 const networks = require('./networks')
 
+/**
+ * @param network
+ * @returns {string} the name of the network. Returns undefined if network is not a value
+ *                   of `networks`
+ */
+function getNetworkName (network) {
+  return Object.keys(networks).find(n => networks[n] === network)
+}
+
+/**
+ * @param network
+ * @returns {Object} the mainnet corresponding to a testnet
+ */
+function getMainnet (network) {
+  switch (network) {
+    case networks.bitcoin:
+    case networks.testnet:
+      return networks.bitcoin
+
+    case networks.bitcoincash:
+    case networks.bitcoincashTestnet:
+      return networks.bitcoincash
+
+    case networks.bitcoingold:
+    // FIXME(https://github.com/BitGo/bitgo-utxo-lib/issues/50): define bitcoingoldTest
+      return networks.bitcoingold
+
+    case networks.bitcoinsv:
+    case networks.bitcoinsvTestnet:
+      return networks.bitcoinsv
+
+    case networks.dash:
+    case networks.dashTest:
+      return networks.dash
+
+    case networks.litecoin:
+    case networks.litecoinTest:
+      return networks.litecoin
+
+    case networks.zcash:
+    case networks.zcashTest:
+      return networks.zcash
+  }
+  throw new TypeError(`invalid network`)
+}
+
+/**
+ * @param network
+ * @returns {boolean} true iff network is a mainnet
+ */
+function isMainnet (network) {
+  return getMainnet(network) === network
+}
+
+/**
+ * @param network
+ * @returns {boolean} true iff network is a testnet
+ */
+function isTestnet (network) {
```

Comment:
This made me look twice first, because we've been burned by catch-all defaults before, but one second thought it's reasonable to assume that there is only one mainnet and everything else is some form of testnet.

I also considered what would happen if one put an invalid network, but realized that this would cause `getMainnet(…)` to throw.